### PR TITLE
Md Files

### PR DESCRIPTION
**Pull request by Shubhangi-cs**

### CODE REVIEW

File: docs/Salesforce-connector.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+----------
+**Name:** Name of the connection. Connection names must be unique in a namespace.
+
+**Description:** Description of the connection.
+
+**Username:** Salesforce username.
+
+**Password:** Salesforce password.
+
+**Security Token:** Salesforce security Token. If the password does not contain the security token the plugin
```
Comment (robinrielley) [inline docs/Salesforce-connector.md]: Can we add a comma "If the password does not contain a security token, "

File: docs/Salesforce-connector.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+will append the token before authenticating with salesforce.
```
Comment (robinrielley) [inline docs/Salesforce-connector.md]: Change salesforce to Salesforce

File: docs/Salesforce-batchsink.md [hunk 3]
```diff
@@ -32,6 +32,12 @@ A Salesforce connected application must be created in order to get a client secr
 
 **SObject Name:** Salesforce object name to insert records into.
 
+**Use Connection:** Whether to use a connection. If a connection is used, you do not need to provide the credentials.
+
+**Connection:** Name of the connection to use. Object Names information will be provided by the connection.
+You also can use the macro function ${conn(connection-name)}.
```
Comment (robinrielley) [inline docs/Salesforce-batchsink.md]: The .md has a problem. The example for SObject Name is under the Connection property now.

File: docs/Salesforce-batchsink.md [hunk 4]
```diff
@@ -32,6 +32,12 @@ A Salesforce connected application must be created in order to get a client secr
...
+
+
```
Comment (robinrielley) [inline docs/Salesforce-batchsink.md]: I think the SObject Name property got deleted?

File: docs/Salesforce-batchsource.md [hunk 5]
```diff
@@ -33,6 +33,12 @@ A Salesforce connected application must be created in order to get a client secr
 
 **SOQL Query:** A SOQL query to fetch data into source.
 
+**Use Connection:** Whether to use a connection. If a connection is used, you do not need to provide the credentials.
+
+**Connection:** Name of the connection to use. Object Names information will be provided by the connection.
+You also can use the macro function ${conn(connection-name)}.
+
```
Comment (robinrielley) [inline docs/Salesforce-batchsource.md]: Same problem as with the Salesforce sink .md. SObject Name seems to have been removed and the example is under the Connection property.

File: docs/SalesforceMultiObjects-batchsource.md [hunk 6]
```diff
@@ -29,6 +29,10 @@ A Salesforce connected application must be created in order to get a client secr
 
 **Login Url:** Salesforce OAuth2 login url.
 
+**Use Connection:** Whether to use a connection. If a connection is used, you do not need to provide the credentials.
+
+**Connection:** Name of the connection to use.
```
Comment (robinrielley) [inline docs/SalesforceMultiObjects-batchsource.md]: Same issue here. The SObject Name examples are now under Connection property.

File: docs/Salesforce-connector.md (hunk 1 above)
Comment (Shubhangi-cs) [reply docs/Salesforce-connector.md]: Done

File: docs/Salesforce-connector.md (hunk 2 above)
Comment (Shubhangi-cs) [reply docs/Salesforce-connector.md]: Done

File: docs/Salesforce-batchsink.md (hunk 3 above)
Comment (Shubhangi-cs) [reply docs/Salesforce-batchsink.md]: Changed the sequence.

File: docs/Salesforce-batchsink.md (hunk 4 above)
Comment (Shubhangi-cs) [reply docs/Salesforce-batchsink.md]: No, pls check again.

File: docs/Salesforce-batchsource.md (hunk 5 above)
Comment (Shubhangi-cs) [reply docs/Salesforce-batchsource.md]: Changed the sequence.

File: docs/SalesforceMultiObjects-batchsource.md (hunk 6 above)
Comment (Shubhangi-cs) [reply docs/SalesforceMultiObjects-batchsource.md]: Changed the sequence.

File: docs/Salesforce-batchsink.md [hunk 7]
```diff
@@ -17,7 +17,7 @@ Configuration
 
 **Password:** Salesforce password.
 
-**Security Token:** Salesforce security Token. If the password does not contain the security token the plugin 
+**Security Token:** Salesforce security Token. If the password does not contain the security token, the plugin 
 will append the token before authenticating with salesforce.
```
Comment (robinrielley) [inline docs/Salesforce-batchsink.md]: Can you change salesforce to Salesforce?

File: docs/Salesforce-streamingsource.md [hunk 8]
```diff
@@ -25,7 +25,7 @@ Configuration
 
 **Password:** Salesforce password.
 
-**Security Token:** Salesforce security Token. If the password does not contain the security token the plugin 
+**Security Token:** Salesforce security Token. If the password does not contain the security token, the plugin 
 will append the token before authenticating with salesforce.
```
Comment (robinrielley) [inline docs/Salesforce-streamingsource.md]: Can you change salesforce to Salesforce?

File: docs/SalesforceMultiObjects-batchsource.md [hunk 9]
```diff
@@ -18,7 +18,7 @@ Configuration
 
 **Password:** Salesforce password.
 
-**Security Token:** Salesforce security Token. If the password does not contain the security token the plugin 
+**Security Token:** Salesforce security Token. If the password does not contain the security token, the plugin 
 will append the token before authenticating with salesforce.
```
Comment (robinrielley) [inline docs/SalesforceMultiObjects-batchsource.md]: Can you change salesforce to Salesforce?

File: docs/Salesforce-batchsource.md [hunk 10]
```diff
@@ -18,7 +18,7 @@ Configuration
 
 **Password:** Salesforce password.
 
-**Security Token:** Salesforce security Token. If the password does not contain the security token the plugin 
+**Security Token:** Salesforce security Token. If the password does not contain the security token the plugin, 
 will append the token before authenticating with salesforce.
```
Comment (robinrielley) [inline docs/Salesforce-batchsource.md]: Can you change salesforce to Salesforce?

**Review by robinrielley — CHANGES_REQUESTED** [review]
Can you change salesforce to Salesforce in the last word of the description for this property: Security Token: Salesforce security Token. If the password does not contain the security token, the plugin will append the token before authenticating with salesforce.

File: docs/Salesforce-batchsink.md (hunk 7 above)
Comment (Shubhangi-cs) [reply docs/Salesforce-batchsink.md]: Done

File: docs/Salesforce-streamingsource.md (hunk 8 above)
Comment (Shubhangi-cs) [reply docs/Salesforce-streamingsource.md]: Done

File: docs/SalesforceMultiObjects-batchsource.md (hunk 9 above)
Comment (Shubhangi-cs) [reply docs/SalesforceMultiObjects-batchsource.md]: Done

File: docs/Salesforce-batchsource.md (hunk 10 above)
Comment (Shubhangi-cs) [reply docs/Salesforce-batchsource.md]: Done

**Pull request closed without merging**